src/components/constants.js

```diff
@@ -8,8 +8,16 @@ export const API_END_POINTS = {
 export const BAERER = 'Bearer';
 export const TOKEN_KEY = 'TOKEN_KEY';
 
+
 // Labes //
+export const HOME_LABEL = 'Home'
+export const ABOUT_LABEL = 'About'
```

Comment:
you can probably separate your page-specific variables tp their own files if they're not going to be used in different spots in the app. This will make it easier to keep track of them. That's optional though.

Reply:
Тhese and some other variables were transferred to the relevant pages. @virzhinia-petkova-redshelf